app/kumactl/cmd/completion/completion_test.go

```diff
@@ -50,7 +50,7 @@ var _ = Describe("kumactl completion", func() {
 			// when
 			actual := stdout.Bytes()
 			// then
-			Expect(len(actual)).To(Equal(len(expected)))
+			Expect(len(actual)).To(Equal(len(expected)), "run kumactl completion bash/fish/zsh, copy the output and override testdata/*.golden")
```

Comment:
Please use `Expect(actual).To(HaveLen(len(expected)))`. In case of failing output will be different, compare:
```
  Expected
      <int>: 3
  to equal
      <int>: 2
```
and
```
  Expected
      <[]int | len:3, cap:3>: [1, 2, 3]
  to have length 2
```

Reply:
Will change in the next PR.